core/escrow/escrow.go

```diff
@@ -2,14 +2,14 @@ package escrow
 
 import (
 	"context"
-	"encoding/base64"
 	"fmt"
 	"time"
 
 	config "github.com/TRON-US/go-btfs-config"
 	"github.com/gogo/protobuf/proto"
 	ic "github.com/libp2p/go-libp2p-core/crypto"
 	"github.com/tron-us/go-btfs-common/crypto"
+	ccrypto "github.com/tron-us/go-btfs-common/crypto"
```

Comment:
Why there is two same "github.com/tron-us/go-btfs-common/crypto" here? and name the second one to ccrypto？ Am I missing something?

Reply:
Didn't see there is already one, just removed ccrypto.